website/docs/07-apps/01-included-in-onion/pico8-wrapper.md

```diff
@@ -0,0 +1,115 @@
+---
+slug: /apps/pico8-wrapper
+description: PICO-8 Native Wrapper by XK9274
+---
+
+:::caution 
+This section refers to an application planned for inclusion in a future Onion release. Final instructions and usage details may differ from the ones described here. 
+Check [here](https://www.reddit.com/r/MiyooMini/comments/17nw79r/announcing_native_pico8_wrapper_for_onion_with/k7ueyec/?context=3) for playtesting details.
+:::
+
+# PICO-8 Native Wrapper / Splore
+
+The PICO-8 Native Wrapper allows you to play PICO-8 without the need for emulation (hence providing full compatibility with all PICO-8 games), and grants access to Splore, a "frontend" app that allows you to browse, search and play through the entirety of the PICO-8 catalogue (as indexed by the [Lexaloffle BBS](https://www.lexaloffle.com/bbs/)), as long as you have an internet connection. 
+
+Splore also allows you to navigate through and play local cartridge files.
+
+This is an alternative to the RetroArch `fake-08` core, which has access to all typical RetroArch features such as rewinding, fast forward and save state management, but can have compatibility with a portion of the PICO-8 library, specially recently released games.
+
+## How to Enable and Launch
+
+1. Enable the `PICO-8 Native Wrapper` App using the Onion Package Manager's *Expert* section.

```

Comment:
It will show as "PICO-8 (PICO-8 standalone)" 

![PackageManager_000](https://github.com/OnionUI/Onion/assets/47260768/0f0ef8eb-1bb2-475c-9b1c-1ad19cf2d81f)